README.md

```diff
@@ -103,6 +105,57 @@ function ExtensibleApp({ params, auth, shell, service, version, env }) {
 ```
 
 Components inside the Extensible provider can use the `useExtensions` hook.
+Basic usage could looks like:
+
+#### `Component.jsx`
+```jsx
+import React from 'react';
+import { useExtensions } from '@adobe/uix-host-react'
+
+export default function Component() {
+  const [extensionMsg, setExtensionMsg] = useState('');
+
+
+  const { extensions = {} } = useExtensions(() => ({
+    requires: {
+      extensionNamespace: ["getMessage"]
+    },
+    provides: {
+      hostNamespace: {
+        getHostInfo: () => {
+          return `Message from the host to guest!`;
+        },
+      },
+    }
+  }));
+
+
+  return (
+    <div>
+      {extensions[0] && (
+        <GuestUIFrame key={Math.random()} guestId={extensions[0].id} src={extensions[0].url.href}/>
```

Comment:
- We can use extension id as a key instead of random. 
- extensions is the list, not object ` const { extensions = {} } = useExtensions(() => ({`
- Probably we should split this example for like 2 components. We usually `useExtensions` in hooks that represent specific extension points, like `useRailExtensions`, `useButtonsExtensions`. That hooks return data of specific extensions, but not  HTML/ReactDom. For transferring data to UI elements (react components), we use renderers like `ActionBarItemsRenderer` or, in the case of GuestUIFrame, usually renders directly inside the application source code with a prepared selected extension that should be rendered at this moment  (extensions[0] - isn't the best example, need to provide some simple example of selection logic)
- I suggest splitting this example into subcategories, host/main application, and extension application.  the example below `const guestServer = await register({` is a part of extension. The example above is part of host/main application but thats unclear from documentation.